environments/sbox/sbox.tfvars

```diff
@@ -18,7 +18,7 @@ cdn_sku    = "Standard_Verizon"
 shutter_rg = "shutter-app-sbox-rg"
 
 frontend_agw_private_ip_address = "10.2.13.114"
-cft_apps_cluster_ips            = ["10.2.11.250"]
+cft_apps_cluster_ips            = ["10.2.9.250"]
 
```

Comment:
It might be better to add 00 back in and then remove 01. Don't think it'll generally be an issue but we might want to encourage doing it that way in case the pipeline fails and we end up with no IPs added to the LB in higher environments